scipy/stats/_unuran/unuran_wrapper.pyx.templ

```diff
@@ -327,6 +335,8 @@ cdef void _pack_distr(unur_distr *distr, dict callbacks) except *:
             unur_distr_cont_set_dpdf(distr, dpdf_thunk)
         if "cdf" in callbacks:
             unur_distr_cont_set_cdf(distr, cont_cdf_thunk)
+        if "logpdf" in callbacks:
+            unur_distr_cont_set_logpdf(distr, logpdf_thunk)
```

Comment:
Is this where the magic happens? I feel like I'm missing the code that would actually use the `logpdf` to do something, but my guess is that you just tell UNURAN that it's available here, and it will use it if it so chooses (e.g. if `pdf` is not specified)?

Reply:
> Is this where the magic happens? I feel like I'm missing the code that would actually use the `logpdf` to do something, but my guess is that you just tell UNURAN that it's available here, and it will use it if it so chooses (e.g. if `pdf` is not specified)?

Yes. If the PDF is not found, it will choose logpdf. If both are present, I think PDF is used by default. But if both methods are present (e.g. SciPy distributions), it would be better to add another parameter `islog` which specifies which method to use. What do you think @mdhaber, @chrisb83?

Reply:
I was thinking to leave it as it is for now. At the moment, there is no specific advantage to use pdf vs logpdf, it is mainly for convenience. So we can let UNU.RAN decide internally and if the user wants to specifically use one method, a custom object with only pdf / logpdf has to be defined

Reply:
I guess it's fine to keep it as-is. But I think we should mention it in the docs. Something along the line "if both pdf and logpdf is specified, pdf is used." What do you think?

Reply:
done

Reply:
Thanks for looking into this @chrisb83!